### ISSUE TITLE
Content links should be updated

### ISSUE THREAD

**Issue by Panquesito7** [mt-mods/moretrees#31]
### Description

The [ContentDB page](https://content.minetest.net/packages/mt-mods/moretrees/) links to the mod's old links, which VanessaE previously owned.
These links should be updated for easier accessibility and to prevent 404 pages.

**Comment by Niklp09** [comment]
idk who changed the links, but I can't find any links to GitLab in the page source

**Comment by wsor4035** [comment]
checked, nothing changed recently, noticed issue tracker went to the a super ancient link, so fixed that. @Panquesito7 please be more specific next time/comment specifically if the issue hasnt been addressed
